@rollup/plugin-babel incorrectly listed as direct dependency

**Description**
Following commit https://github.com/djipco/webmidi/commit/ac627227584db82d1f113ced98b247b4ed3ec87f, `@rollup/plugin-babel` is listed as a direct dependency rather than a `devDependency`.
This causes installation warnings or errors depending on the package manager used.
For instance, using `pnpm`:
```(bash)
 ERR_PNPM_PEER_DEP_ISSUES  Unmet peer dependencies
.
└─┬ webmidi
  └─┬ @rollup/plugin-babel
    ├── ✕ missing peer rollup@^1.20.0||^2.0.0
    ├── ✕ missing peer @babel/core@^7.0.0
    └─┬ @rollup/pluginutils
      └── ✕ missing peer rollup@^1.20.0||^2.0.0
Peer dependencies that should be installed:
  @babel/core@^7.0.0                          rollup@">=1.20.0 <2.0.0 || >=2.0.0 <3.0.0"
```

This is easily fixed by moving it back to the `devDependencies`.

--- a/package.json
+++ b/package.json
@@ -81,7 +81,6 @@
     "test:WebMidi": "mocha test/WebMidi.test.js"
   },
   "dependencies": {
-    "@rollup/plugin-babel": "^5.3.0",
     "djipevents": "^2.0.5"
   },
   "optionalDependencies": {
@@ -95,6 +94,7 @@
     "@babel/polyfill": "^7.8.7",
     "@babel/preset-env": "^7.9.0",
     "@octokit/graphql": "^4.8.0",
+    "@rollup/plugin-babel": "^5.3.0",
     "array-back": "^6.2.0",
     "babel-plugin-istanbul": "^6.0.0",
     "chai": "^4.3.3",

